sum.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+
+int main()
+{
+	int x, y, counter,i;
+	printf("Enter how long you would like the array to be: ");
+	scanf("%d\n",&x);
+	int sum[x];
+	for(i=0;i<x;i++)
+	{
+		printf("Enter a number: ");
+		scanf("%d", &sum[i]);
+	}
+	int new[x];
+	for(y=0;y<x;y++)
+	{
+		counter= counter+ sum[y];
+		new[y]=counter;
+	}
+	for(y=0;y<x;y++)
```

Comment:
Not sure why you did this for loop two times - you should have only done it once. Once you fix that, your code works as needed! Also, make sure to include comments